### PR TITLE
Jetpack: Ver bump to 9.8-alpha

### PR DESCRIPTION
**Pull request by kraftbj**
- Bump version
- Changelog

Bumps version.

#### Changes proposed in this Pull Request:

* ver bump

#### Jetpack product discussion

n/a

#### Does this pull request change what data or activity we track or use?

no

#### Testing instructions:

* Go to '..'
*

### CODE REVIEW

File: projects/plugins/jetpack/changelog/add-9.8-alpha [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: other
+
+bump ver number
```
Comment (anomiex) [inline projects/plugins/jetpack/changelog/add-9.8-alpha:4]: No need for a changelog entry for this, IMO.
```suggestion
Comment: bump ver number

```

Comment (kraftbj) [reply projects/plugins/jetpack/changelog/add-9.8-alpha]: Did the CLI fast and didn't read the prompt I guess!

**Comment by sdixon194** [comment]
Does `composer.json` also need to be bumped, as well as the header in `jetpack.php`?

**Comment by kraftbj** [comment]
> Does `composer.json` also need to be bumped, as well as the header in `jetpack.php`?

I didn't realize about the autoloader suffix and just forgot about the plugin headers.

**Review by anomiex — APPROVED** [review]
That's why we have a script to do the update ;)

**Comment by kraftbj** [comment]
> That's why we have a script to do the update ;)

:oldmanyellsatcloud:

**Comment by kraftbj** [comment]
r224878-wpcom

**Pull request merged**